ifrs17/Report/ReportScopes.ipynb

```diff
@@ -269,6 +269,7 @@
             "source": [
                 "public interface BestEstimate: IScope<(ReportIdentity Id, CurrencyType CurrencyType), ReportStorage>, IDataCube<ReportVariable> {",
                 "\n    IDataCube<ReportVariable> BestEstimate => Identity.Id switch {",
+                "\n            { ValuationApproach: ValuationApproaches.PAA } => GetScope<LockedBestEstimate>(Identity).LockedBestEstimate,",
```

Comment:
add a todo here to use the economic basis driver to decide which Economic basis to use.